SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -15,6 +15,18 @@ void USpatialDispatcher::Init(USpatialNetDriver* InNetDriver)
 	NetDriver = InNetDriver;
 	Receiver = InNetDriver->Receiver;
 	StaticComponentView = InNetDriver->StaticComponentView;
+
+	// Collect all OpCallbackTemplate subclasses to register user callbacks in pipeline.
+	for (TObjectIterator<UClass> OpCallbackClass; OpCallbackClass; ++OpCallbackClass)
```

Comment:
```suggestion
	for (TObjectIterator<UClass> OpCallbackClass; OpCallbackClass != nullptr; ++OpCallbackClass)
```

Reply:
This refused to compile, will post the actually error it caused when I get back in tmmo

Reply:
Unreal iterators override their `operator bool`, the code is perfectly fine as it is.